Autopilot/Inc/airspeed.hpp

```diff
@@ -69,11 +70,15 @@ class MPXV7002DP : public airspeed {
         void GetResult(airspeedData_t &Data);
 
         /**
-         *  Triggers interrupt for new airspeed measurement - stores 
-         *  raw data in variables. This function is blocking, as polling for the adc has
-         *  a timeout of 50 ms.
+         *  Don't use this function, the GetResult function is non blocking.
          */
         void Begin_Measuring();
```

Comment:
Can you please write it as `void Begin_Measuring() {}`. That way it is defined as empty